frontend/model/contracts/group.js

```diff
@@ -264,7 +216,7 @@ DefineContract({
         const memberProfile = getters.currentGroupState.profiles[username]
         if (!memberProfile) return false
         const memberJoinedMs = new Date(memberProfile.joinedDate).getTime()
-        const joinedAfterUs = weJoinedMs < memberJoinedMs
+        const joinedAfterUs = weJoinedMs <= memberJoinedMs
```

Comment:
Note: `groupMembersSorted` must also be moved to `state.js`. Notice it uses `Date.now()` below. That's a clear sign that dynamic code is being used — data not derived from the contract messages — so this entire function must be moved out of this contract getter.

Reply:
Moving it...